python/cugraph/traversal/bfs.py

```diff
@@ -106,14 +101,15 @@ def _convert_df_to_output_type(df, input_type):
             else:
                 return (distances, preds)
     else:
-        raise TypeError(f"input type {input_type} is not a supported type.")
+        raise TypeError("input type {input_type} is not a supported type.")
 
 
 def bfs(G,
         start=None,
-        return_sp_counter=None,
+        depth_limit=None,
```

Comment:
This needs to be added to the docstring.

Reply:
Its WIP, still need to clean up code

Reply:
added to docstring